Common spacing in components

antidecaf opened the pull request:
This PR replaces hardcoded spacing values with universal variables defined in [`ffe-core`](https://github.com/SpareBank1/designsystem/blob/develop/packages/ffe-core/less/spacing.less). The values are based on the [8pt grid](https://builttoadapt.io/intro-to-the-8-point-grid-system-d2573cde8632) spatial system.

For the most part, hardcoded values could be replaced by a more or less equivalent variable. Some values could have potentially been replaced by several different variables (e.g. do I replace 20px with 16px or 24px?). I've tried to pick the ones that make the most sense on a case by case basis.

This _will_ cause visual diffs all over the place. :see_no_evil:

antidecaf committed:
fix(ffe-accordion): replace hardcoded spacing with variables

BREAKING CHANGE: This commit is likely to cause small visual regressions caused by slightly altered margin and padding values.

antidecaf committed:
fix(ffe-buttons): replace hardcoded spacing with variables

BREAKING CHANGE: This commit is likely to cause small visual regressions caused by slightly altered margin and padding values.

antidecaf committed:
fix(ffe-cards): replace hardcoded spacing with variables

BREAKING CHANGE: This commit is likely to cause small visual regressions caused by slightly altered margin and padding values.

antidecaf committed:
fix(ffe-context-message): replace hardcoded spacing with variables

BREAKING CHANGE: This commit is likely to cause small visual regressions caused by slightly altered margin and padding values.

antidecaf committed:
fix(ffe-datepicker): replace hardcoded spacing with variables

BREAKING CHANGE: This commit is likely to cause small visual regressions caused by slightly altered margin and padding values.

antidecaf committed:
fix(ffe-file-upload): replace hardcoded spacing with variables

BREAKING CHANGE: This commit is likely to cause small visual regressions caused by slightly altered margin and padding values.

antidecaf committed:
fix(ffe-form): replace hardcoded spacing with variables

BREAKING CHANGE: This commit is likely to cause small visual regressions caused by slightly altered margin and padding values.

antidecaf committed:
fix(ffe-grid): replace hardcoded spacing with variables

BREAKING CHANGE: This commit is likely to cause small visual regressions caused by slightly altered margin and padding values.

antidecaf committed:
fix(ffe-header): replace hardcoded spacing with variables

BREAKING CHANGE: This commit is likely to cause small visual regressions caused by slightly altered margin and padding values.

antidecaf committed:
fix(ffe-lists): replace hardcoded spacing with variables

BREAKING CHANGE: This commit is likely to cause small visual regressions caused by slightly altered margin and padding values.

antidecaf committed:
fix(ffe-message-box): replace hardcoded spacing with variables

BREAKING CHANGE: This commit is likely to cause small visual regressions caused by slightly altered margin and padding values.

antidecaf committed:
fix(ffe-system-message): replace hardcoded spacing with variables

BREAKING CHANGE: This commit is likely to cause small visual regressions caused by slightly altered margin and padding values.

antidecaf committed:
fix(ffe-tables): replace hardcoded spacing with variables

BREAKING CHANGE: This commit is likely to cause small visual regressions caused by slightly altered margin and padding values.

antidecaf committed:
fix(ffe-tabs): replace hardcoded spacing with variables

BREAKING CHANGE: This commit is likely to cause small visual regressions caused by slightly altered margin and padding values.

ivarni commented:
Great to see this moving forward!

I'm wondering if peerDeps around the board can just be changed from `x` to `> x`  just to pre-empt the inevitable chaos releasing a major of every package usually unleashes? :)

antidecaf committed:
fix(ffe-account-selector-react): replace hardcoded spacing with vars

BREAKING CHANGE: This commit is likely to cause small visual regressions caused by slightly altered margin and padding values.

antidecaf commented:
@ivarni Sounds like a good idea to me, but I'm not sure how to work that out with Lerna.

ivarni commented:
Fair point, nothing seems to work out very well with Lerna :)
